crates/top/re_sdk/src/lenses/ast.rs

```diff
@@ -846,4 +893,281 @@ mod test {
         assert_eq!(my_timeline.times_raw()[3], 130);
         assert_eq!(my_timeline.times_raw()[4], 140);
     }
+
+    // Helper function to create test data: list of structs with {timestamp: i64, value: String}
+    fn create_test_struct_list() -> ListArray {
```

Comment:
Since you log errors instead of returning them, I think these tests should add a hook-in to ensure that no errors were logged during their runtime